src/components/course-settings/roster.cjsx

```diff
@@ -14,10 +14,27 @@ module.exports = React.createClass
   propTypes:
     courseId: React.PropTypes.string.isRequired
 
+  updateCourseName:(courseId)->
+    RosterActions.save(@props.courseId)
+    console.log('waka kinda working');
+
+  changeCourseName:->
+    course = CourseStore.get(@props.courseId)
+    <BS.Popover tilte={'Change Course Name'} {...@props}>
```

Comment:
is `...@props` needed? It seems odd that the props sent to the `<Roster>` would be passed into the popover